### DISCUSSION TITLE
Support PADES electronic signature.

### DISCUSSION THREAD

**Discussion by vperilla** [docusealco/docuseal#89] [Ideas]
Is there a plan to support electronic signature using PADES standard?
https://en.wikipedia.org/wiki/PAdES

**Comment by AlexBTurchyn** [comment]
@vperilla I need to research a bit more about PAdES. As far as I know (AFAIK), it requires sending a hex digest to a trusted authority, which makes it not viable when self-hosting. However, I hope that Advanced Electronic Signature will eventually be available on docuseal.co cloud.

**Reply by ovizii** [reply]
I have only just started reading up on electronic signatures but now that you mention "Advanced Electronic Signature" I assume that right now, Docuseal only offers a "Simple Electronic Signature"? 

I have no needs for this, just trying to read up on how this works (legally). So far I have found this overview. Maybe you could try and add some of this info on Docuseal's homepage.

https://ec.europa.eu/digital-building-blocks/wikis/display/ESIGKB/What+are+the+levels%2C+simple%2C+advanced+and+qualified+of+electronic+signatures

![image](https://github.com/docusealco/docuseal/assets/837902/cc0d7ba0-f1f2-4a5f-b87e-6d4395784f57)

**Reply by AlexBTurchyn** [reply]
@ovizii, thank you for sharing this. You're right, at DocuSeal, we are fully compliant only with the first Simple Electronic signature based on the eIDAS standard.

The advanced level with PADES digital signature can be achieved with DocuSeal Cloud, and we hope to introduce this level of signatures soon.

**Comment by AlexBTurchyn** [comment]
@vperilla Yes, we plan to be fully compliant with the PADES standard.

**Reply by dazzag24** [reply]
Hi, 

Can you let me know if this has been added yet?  I'm doing vendor selection at the moment and this is a requirement.
